src/index.js

```diff
@@ -27,7 +36,10 @@ class HookActions {
  *          actions that can be invoked.
  */
 export function useGovernor(initialState = {}, contract = {}) {
-  if (!contract || typeof contract !== "object") {
+  if (
+    !contract ||
+    (typeof contract !== "object" && typeof contract !== "function")
```

Comment:
This will allow:

```
function myContract() {
  function increment(state) {
    return {
      count: state.count + 1
    };
  }
}
```

This will make it all the way through without an error, but instead they'll have a contract with no keys. If we want to be really pedantic, we should test that the function is indeed a class.

```
  if (
    !contract ||
    (typeof contract !== "object" && typeof contract !== "function") ||
    // If a function type has an arguments property, it's not a class.
    (typeof contract === "function" && Object.getOwnPropertyNames(contract).includes("arguments")) {
```

Reply:
Determining what is and what is not a `class` is harder than that. I'm not doing this because it's so little benefit; if someone passes a function with a closure inside and expects it to work (and it won't), they have bigger problems.